### PR TITLE
feat: Create App Registry application for SWB

### PR DESCRIPTION
**Pull request by FernandoAranda**
Issue #, if available:

Description of changes:
Add App Registry on backend deploy

Checklist:

* [x] Have you successfully deployed to an AWS account with your changes?
* [ ] Have you written new tests for your core changes, as applicable?

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: solutions/swb-reference/src/index.ts [hunk 1]
```diff
@@ -4,10 +4,21 @@
...
 
 /* eslint-disable no-new */
+import { WorkbenchAppRegistry } from '@aws/workbench-core-infrastructure';
 import * as cdk from 'aws-cdk-lib';
+import { config } from 'dotenv';
 import { SWBStack } from './SWBStack';
 
-const app: cdk.App = new cdk.App();
-new SWBStack(app);
+config({ path: './src/config/appRegisrty.env' });
```
Comment (aws-tyler) [inline solutions/swb-reference/src/index.ts:12]: `appRegistry` not `appRegisrty`

File: solutions/swb-reference/src/index.ts [hunk 2]
```diff
@@ -4,10 +4,21 @@
  */
 
 /* eslint-disable no-new */
+import { WorkbenchAppRegistry } from '@aws/workbench-core-infrastructure';
 import * as cdk from 'aws-cdk-lib';
+import { config } from 'dotenv';
```
Comment (aws-tyler) [inline solutions/swb-reference/src/index.ts:9]: i'm glad to see we're using `dotenv` but why do we need it now? we've been able to read env files before right?

**Review by aws-tyler — APPROVED** [review]

File: solutions/swb-reference/src/index.ts (hunk 2 above)
Comment (FernandoAranda) [reply solutions/swb-reference/src/index.ts:9]: I saw the example of the test application and followed their pattern, I didn't want to have it in yaml as those values are normally modified by users and this values should not be touched by users

Comment (FernandoAranda) [reply solutions/swb-reference/src/index.ts:9]: On a second tought I think this values can be on constants while we figure a way to retrieve them dynamically

**Pull request closed without merging**